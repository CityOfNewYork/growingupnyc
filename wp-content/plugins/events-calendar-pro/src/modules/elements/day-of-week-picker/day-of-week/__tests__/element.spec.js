/**
 * External dependencies
 */
import React from 'react';
import { noop } from 'lodash';

/**
 * Internal dependencies
 */
import DayOfWeek from '../element';

describe( 'Day Of Week Element', () => {
	it( 'renders day of week', () => {
		const component = renderer.create(
			<DayOfWeek
				checked={ true }
				className="test-class"
				label="F"
				labelTitle="Friday"
				onChange={ noop }
				value="friday"
			/>
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
