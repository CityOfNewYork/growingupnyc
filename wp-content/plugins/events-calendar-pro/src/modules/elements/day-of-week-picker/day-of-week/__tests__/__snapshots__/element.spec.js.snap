// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Day Of Week Element renders day of week 1`] = `
<div
  className="tribe-editor__checkbox tribe-editor__day-of-week test-class"
>
  <input
    checked={true}
    className="tribe-editor__input tribe-editor__input--checkbox tribe-editor__checkbox__input"
    id="uniqid"
    onChange={[Function]}
    type="checkbox"
    value="friday"
  />
  <label
    className="tribe-editor__checkbox__label"
    htmlFor="uniqid"
  >
    <abbr
      className="tribe-editor__day-of-week__label-abbr"
      title="Friday"
    >
      F
    </abbr>
  </label>
</div>
`;
